config/ground-truth-yaml-files/all_features_version_10_no_visit_variant_pcd.yml

```diff
@@ -2647,9 +2657,9 @@ patient:
   #  - biolink:PhenotypicFeature
   # type: number
   ur:
-    categories:
-    - biolink:EnvironmentalExposure
-    enum: &id1849
-    - R
-    - U
-    type: string
+   categories:
+   - biolink:EnvironmentalExposure
+   enum: &id1849
+   - R
+   - U
+   type: string

```

Comment:
@karafecho Same as the previous comment. Need to fix the spacing issues here. The old file has the correct spacing here.